packages/webapp/src/utils/register-task-list.js

```diff
@@ -9,7 +9,8 @@ const registerTaskList = {
           title: 'Add your details',
           status: constants.DEFAULT_REGISTRATION_TASK_STATUS,
           completedTaskUrl: constants.routes.CHECK_YOUR_DETAILS,
-          startTaskUrl: constants.routes.NAME
+          startTaskUrl: constants.routes.NAME,
+          id: 'add-your-details'
```

Comment:
What are these ID additions on this script? I don't think they're related to BNGP-1099

Reply:
Yes, they are not related to this ticket. Shaun wanted the ids for automated testing. I included them in this ticker.